src/installCommand/index.js

```diff
@@ -108,9 +109,46 @@ const installationSpec = {
         await applyPatch(packageJson, target);
       }
     },
+
+    installDidComplete: async (installedPackages) => {
+      if (!installedPackages) {
+        return;
+      }
+      await linkConditionalDependencies(installedPackages);
```

Comment:
Right now I'm not sure what other hooks we need from pnpm and in what form — I'm also want to dog food pnpm plugin API a bit before submitting a merge proposal to pnpm.